section2/set9/exercises/04-verbose-mode/main.go

```diff
@@ -0,0 +1,105 @@
+// Copyright © 2018 Inanc Gumus
+// Learn Go Programming Course
+// License: https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// For more tutorials  : https://learngoprogramming.com
+// In-person training  : https://www.linkedin.com/in/inancgumus/
+// Follow me on twitter: https://twitter.com/inancgumus
+
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+// ---------------------------------------------------------
+// EXERCISE: Verbose Mode
+//
+//  When the player runs the game like this:
+//
+//    go run main.go -v 4
+//
+//  Display each generated random number:
+
+//    1 3 4 🎉  YOU WIN!
+//
+//  In this example, computer picks 1, 3, and 4. And the
+//  player wins.
+//
+// HINT
+//  You need to get and interpret the command-line arguments.
+// ---------------------------------------------------------
+
+func main() {
+	const (
+		maxTurns = 5
+		usage    = `Welcome to the Lucky Number Game!
+
+The program will pick %d random numbers.
+Your mission is to guess one of those numbers.
+
+The greater your number is, the harder it gets.
+
+Want to play?`
+	)
+
+	rand.Seed(time.Now().UnixNano())
+
+	args := os.Args[1:]
+	var guess int
+	var err error
+
+	if len(args) < 1 {
+		fmt.Printf(usage, maxTurns)
+		fmt.Println()
+		return
+	}
+
+	if args[0] == "-v" {
+		guess, err = strconv.Atoi(args[1])
```

Comment:
Here there is no check for argument 2 !!

Reply:
-v should be said in the first command line argument given to the command

Reply:
args variable is defined as os.Args[1:], meaning that args[0] is equal to os.Args[2]